### DISCUSSION TITLE
Hyprland? Nothing to hype, just Hyprbug. Dont expect anything. Vaxry is in a weird mode, he is above everything, anything else is bad!

### DISCUSSION THREAD

**Discussion by edgars-cirulis** [hyprwm/Hyprland#7688] [General]
Buggy Experience: The current state of Hyprland seems to be quite unstable and filled with bugs. This impacts the usability significantly, especially for those looking for a reliable compositor.

Graphical Standards and User Feedback: There are concerns that Hyprland does not fully adhere to certain graphical standards. Additionally, it appears that user feedback isn't being adequately addressed. For example, the GitHub repository shows over 1,000 reported issues.

Consistent Blame Attribution: It seems that the maintainer often attributes issues to external factors like Chromium, Electron, NVIDIA, and xwayland. These same setups tend to work without problems on other Wayland compositors.

Given these points, I believe it might be worth considering the temporary or permanent removal of Hyprland from all my laptops and workstations until these issues are properly addressed. (Or will they ever be addressed?)

**Comment by vaxerski** [comment]
this is not a train station, you do not need to announce your departure.

**Comment by edgars-cirulis** [comment]
He He.. Vaxry. Thank you for your response. My intention is not to
"announce a departure" but to provide feedback based on my experience and
the issues I've observed while using Hyprland. I believe it's important for
a project's growth to consider user experiences and feedback, especially
when there are significant bugs or compatibility issues that affect
usability.

**Reply by vaxerski** [reply]
saying "shit no work so many bugs" achieves nothing. I can't clone myself.

**Reply by cjmcguire88** [reply]
I've been using Hyprland for 6 months and have yet to face a single problem of significance. I'm going to guess this is another, "Someone hurt my wittle feelings" attack. All that complaining and didn't actually describe any ACTUAL issue. You don't happen to work for Red Hat do you?

**Reply by edgars-cirulis** [reply]
> I've been using Hyprland for 6 months and have yet to face a single problem of significance. I'm going to guess this is another, "Someone hurt my wittle feelings" attack. All that complaining and didn't actually describe any ACTUAL issue. You don't happen to work for Red Hat do you?

No, I do not work for Red Hat. It's just frustrating that Vaxry always blames Nvidia, XWayland, or other external factors. Take that UV bug in Chromium, for instance; he claims it's a Chromium bug, but never claimed Hyprland codebase skill issue. I've heard the same from others who have stopped using Hyprland and switched to River or Sway/SwayFX because of Vaxry consistent blaming of XWayland, Mesa, Nvidia, Chromium, Electron, etc. Meanwhile, everything seems to work fine on other Wayland compositors.

**Reply by mikeygrier** [reply]
dude he just refactored out a huge piece of middleware, replaced it with code he wrote himself.  there will always be bugs when you do some ninja ass shit like that, I'm freaking amazed there were so few!  

not to mention the fact that bare metal GUI dev can't really have comprehensive unit testing, a lot of stuff has to be tested by users.

it only gets better from here!

**Comment by wxlyyy** [comment]
Yeah, the truth

**Comment by pbosab** [comment]
yup, totally useful feedback. even on r/archlinux where you suggested to remove hyprland from the repo. show us where bad guy vaxry touched you.

**Comment by rice-cracker-dev** [comment]
bro wrote a nothing burger

**Comment by mock1328** [comment]
While I'm not going anywhere from hyprland any time soon, several past releases indeed introduced some breakage and performance issues for me, which made overall experience worse. 
I just hope that now, as the major migartion/refactoring process is complete, the focus will shift for mentioned issues, caused by this migration in order to achieve same level of user experience of previous versions...

**Comment by rayhan-ryu** [comment]
While there are some points to be made, the title and the last paragraph don't help anybody.

It's by nature that the bigger and popular a project is, the more issue people will find and report to Github. Sway has 900 issues and Godot has 5K issues for example and the two are massive projects. It's very hard to clear out all the issue especially when a project this massive is mostly done by a single person and few contributors.

**Comment by Aqa-Ib** [comment]
Get a life. Vaxry is giving his best at programming hyprland as free software. You do not respect, look at yourself before talking.

**Comment by rossvold** [comment]
Hyprland has only gotten better and better, and i'm sure that it will keep getting better for a long time in the future.

I'm able to stream, play games, and even do all my daily work on hyprland. And i'm even using an nvidia gpu... 
Seems pretty stable to me.

Huge respect for Vaxry for creating this amazing project out of his own goodwill to open source.

**Comment by sansroot** [comment]
I'm a big fan of the idea behind Hyprland, but for me at least, I've never been able to get it working as well as other X11 tiling window managers. Even though I love new tech, I think Hyprland would have been even better if it had used X11. Anyway, I hope the project keeps going strong and that we see something amazing come out of it someday!

**Reply by vaxerski** [reply]
https://github.com/hyprwm/Hypr

**Comment by odinexdx** [comment]
This can only mean that Hyprland is very popular and nothing else.^.^~!!!!!

**Comment by vdawg-git** [comment]
Why is this still open lol

**Reply by edgars-cirulis** [reply]
Maybe still a thing. :D
